<launch>
  <arg name="base_frame" default="base_link" />
  <arg name="camera_frame1" default="fisheye" />
  <arg name="timeout" default="10" />
  <arg name="tag_frames" default="['tag_584', 'tag_582', 'tag_580',
                                   'tag_579', 'tag_578', 'tag_577',
                                   'tag_9', 'tag_5', 'tag_7']" />

  <param name="/use_sim_time" value="true"/>

  <node pkg="joint_calibration" type="artificial_tfs.py"
        name="artificial_tfs" output="screen"/>

</launch>
